examples/batch/README.md

```diff
@@ -0,0 +1,16 @@
+# Azure Batch Sample
+
+Sample to deploy a Job in Azure Batch
+
+## Creates
+
+1. A Resource Group
+2. A [Storage Account](https://docs.microsoft.com/en-us/azure/batch/batch-api-basics#azure-storage-account)
+3. A [Batch Account](https://docs.microsoft.com/en-us/azure/batch/batch-api-basics#account)
+4. A [Pool](https://docs.microsoft.com/en-us/azure/batch/batch-api-basics#pool) of compute nodes
```

Comment:
since this isn't present in the example yet, can we remove this for the moment?

Reply:
Done.